pnr/openroad/meta.yaml

```diff
@@ -41,16 +41,18 @@ requirements:
     - python
     - libboost 1.73
     - eigen 3.3
-    - spdlog 1.8.1
+    - spdlog 1.9
```

Comment:
>=1.9.2?

Reply:
Is 1.9.2 available on all platforms? I thought I could just specify 1.9 and it would pick any version that was available on that platform. If 1.9.2 is universally available, I'll switch it to that.

Reply:
Looks like 1.9.0 and 1.9.2 are available: so 1.9 sounds good then!

Reply:
https://anaconda.org/main/spdlog/files